provider/redis.go

```diff
@@ -117,9 +121,13 @@ func (table redisOnlineTable) Get(entity string) (interface{}, error) {
 		result, err = val.Float64()
```

Comment:
should it return float32 when it's a float 32 rather than float64?

Reply:
Unsure what you mean. This case is Float64 returning Float64